apps/arena/lib/arena/matchmaking/pair_mode.ex

```diff
@@ -53,23 +53,54 @@ defmodule Arena.Matchmaking.PairMode do
   @impl true
   def handle_info(:launch_game?, %{clients: clients} = state) do
     Process.send_after(self(), :launch_game?, 300)
+
+    state =
+      if Map.has_key?(state, :game_mode_configuration) do
+        state
+      else
+        case Arena.Configuration.get_game_mode_configuration("pair", "battle_royale") do
+          {:error, _} ->
+            state
+
+          {:ok, game_mode_configuration} ->
+            # This is needed because we might not want to send a request every 300 seconds to the game backend
+            Process.send_after(self(), :update_params, 5000)
+            Map.put(state, :game_mode_configuration, game_mode_configuration)
+        end
+      end
+
     diff = System.monotonic_time(:millisecond) - state.batch_start_at
 
-    if length(clients) >= Application.get_env(:arena, :players_needed_in_match) or
-         (diff >= Utils.start_timeout_ms() and length(clients) > 0) do
+    if Map.has_key?(state, :game_mode_configuration) &&
+         (length(clients) >= state.game_mode_configuration.amount_of_players or
+            (diff >= Utils.start_timeout_ms() and length(clients) > 0)) do
       send(self(), :start_game)
     end
 
     {:noreply, state}
   end
 
   def handle_info(:start_game, state) do
-    {game_clients, remaining_clients} = Enum.split(state.clients, Application.get_env(:arena, :players_needed_in_match))
-    create_game_for_clients(game_clients)
+    {game_clients, remaining_clients} = Enum.split(state.clients, state.game_mode_configuration.amount_of_players)
+    create_game_for_clients(game_clients, state.game_mode_configuration)
 
     {:noreply, %{state | clients: remaining_clients}}
   end
 
+  def handle_info(:update_params, state) do
```

Comment:
This is needed to update the params every certain amount of time. That way we will avoid to query the backend every millisecond